python/cudf/cudf/tests/test_spilling.py

```diff
@@ -45,17 +45,47 @@
     )
 
 
-def gen_df(target="gpu") -> cudf.DataFrame:
+def std_df(target="gpu") -> cudf.DataFrame:
```

Comment:
```suggestion
def single_column_df(target="gpu") -> cudf.DataFrame:
```

Reply:
And ditto below?

Reply:
Perhaps we can also consider changing `target="gpu"` to `spilled=True | False` -- but that is less important.

Reply:
Renamed `std_df` => `single_column_df` but I like to keep `target="gpu"`. When we add spilling-to-disk, we introduce `target="disk"`.